src/seacat-auth/components/Authz/UncontrolledSwitchWithAuthz.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import { default as UncontrolledSwitch } from '../UncontrolledSwitch';
+import { default as UncontrolledSwitch } from '../../../components/UncontrolledSwitch';
```

Comment:
@Pe5h4 we have components UncontrolledSwitch and ControlledSwitch. They are located in src. Do I need to copy them to seacat-auth too?